python/paddle/vision/transforms/__init__.py

```diff
@@ -12,11 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from . import transforms
-from . import functional
 
 from .transforms import *
 from .functional import *
 
```

Comment:
`__all__` should include *class* implementations,

Reply:
done